src/impls/create.rs

```diff
@@ -38,7 +38,7 @@ where
 {
     fn default() -> Self {
         Self {
-            map: Default::default(),
+            map: HashMap::default(),
             zero: Default::default(),
```

Comment:
The honest answer is that because only hashmap triggered the clippy.
I think the current lint only looks for `std` or `core` traits.

Taking a closer look into the lint, it would be idiomatic to also change zero to N:: default.
added a commit for that.